_projects/it/devops.md

```diff
@@ -0,0 +1,75 @@
+
+Sulla base di queste nuove esigenze l’organizzazione delle infrastrutture IT sta cambiando ed è in atto una vera e propria rivoluzione che impatta radicalmente sui profili, sull’attitudine e sulle competenze degli stakeholder coinvolti.
+
+Questo cambiamento sta dando vita ad una comunità tecnica allargata composta da System Administrator, Network Architect, DevOps, Site Reliability Engineering, Operations Manager che è sempre alla continua ricerca dei modi migliori per costruire e amministrare infrastrutture IT,  mettendo a fattore comune le esperienze acquisite ed aiutando gli altri a fare lo stesso.
+
+La Pubblica Amministrazione (PA) italiana ha avviato un importante processo di trasformazione digitale attraverso il Piano Triennale 2017-2019 che richiede un cambio di paradigma caratterizzato dall’adozione del Cloud e delle metodologie agile. Saranno le competenze della PA nell’ambito di questi processi e tecnologie a determinare il successo di questo rinnovamento tecnologico, organizzativo e culturale.  
+
+Da queste necessità nasce la comunità DevOps dei servizi digitali che avrà il compito di costruire, condividere, promuovere e consolidare le nuove competenze tecnologiche nel settore pubblico per guidare e sostenere la trasformazione digitale del Paese.
+
+### Principi
+
+La comunità è aperta a tutte e tutti i lavoratori, professionisti, esperti, cultori e promotori di tecnologie nel settore IT che vogliano contribuire al mantenimento ed allo sviluppo armonico dell’insieme dei servizi digitali pubblici.
+
+I partecipanti si propongono di:
+
+1. condividere informazioni, soluzioni e competenze utili per mantenere, aggiornare, aumentare l'affidabilità dei sistemi e per automatizzare le procedure;
+2. implementare, promuovere e diffondere l’utilizzo di tecnologie open e open infrastructure nell'ambito della PA;
+3. discutere e proporre standard e regolamenti dei servizi digitali.
+
+
+### A chi si rivolge la comunità?
+
+Potresti essere interessato a partecipare alla comunità se:
+
+-   sei un amministratore di sistema di servizi digitali della PA e vuoi portare i tuoi servizi sul cloud della PA;
+-   devi progettare un nuovo servizio digitale o portare un servizio esistente sul Cloud della PA;
+-   sviluppi soluzioni e/o hai un ruolo di consulenza per soluzioni utilizzate nella infrastruttura di rete e di servizi della PA;
+-   vuoi condividere tecnologie, soluzioni, esperienze o anche semplicemente una tua indicazione/segnalazione utile per lo sviluppo ed il mantenimento dell’infrastruttura IT della PA.
+
+
+### Come contribuire
+
+Per iniziare a contribuire alla comunità e creare la rete che ci siamo prefissi di creare puoi unirti a noi utilizzando i seguenti canali:
+
+-   [Sezione Forum](https://forum.italia.it/c/piano-triennale/data-center-e-cloud)
+-   [Progetti GitHub](https://github.com/italia/cloud-pa)
+-   [Canale Slack](https://developersitalia.slack.com)
```

Comment:
questo link è errato: bisogna puntare a slack.developers.italia.it per l'invito, e alla chat diretta per chi è già iscritto

Reply:
es: https://developers.italia.it/it/daf/